app/assets/javascripts/external-scripts.js

```diff
@@ -1,22 +1,34 @@
 ExternalScripts = {
-  _constructors: {},
+  _classes: {},
   _scripts:[],
   _script_labels:[]
 };
 
-ExternalScripts.notRegistered = function(label) {
+ExternalScripts._notRegistered = function(label) {
   console.error("No external script registered for " + label);
 };
 
-ExternalScripts.externalScriptError = function(e, other) {
+ExternalScripts._externalScriptError = function(e, other) {
   console.group("External Script Error");
   console.error(e);
   console.dir(other);
   console.groupEnd();
 };
 
+/**************************************************************
+@function init
+This method is called to initialize the external scripts
+Called from the LARA partial `_lightweight.html.haml`
+@arg {string} label - the the script identifier.
+@arg {any} ctx - context object {} for the script, includes
+      scriptLabel -- same as label above
+      scriptUrl -- where the script was loaded from.
+      embeddableId -- the embeddableId of the script.
+      config -- author configuration.
+      div -- the invisible div on the page for this script to render.
```

Comment:
This seems to be broken in the current implementation. It's jQuery object called with a wrong selector (ID without #). It seems no existing scripts use it, so I'd simply remove it from docs. Glossary Plugin will use it just for demo purposes, before we switch to the new Plugin API.

In the future version, I'd rather pass visible div if a plugin is a page-level plugin. So it can stay invisible (by not rendering anything) or actually render something. Or simply don't pass any container it at all, as for now we don't have any use case for it, so it doesn't make sense to guess requirements.

Reply:
You are right.  

I had thought that it was in use by this [model-feedback script](https://github.com/concord-consortium/model-feedback/blob/master/src/lara-approved-scripts/feedback-view.ts)  -- but I see now that  it is not.